### DISCUSSION TITLE
v0.1.700 Preview

### DISCUSSION THREAD

**Discussion by DamianMorozov** [DamianMorozov/OpenTgResearcher#27] [Announcements]
### Added
- Filters settings

<hr /><em>This discussion was created from the release <a href='https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.1.700'>v0.1.700 Preview</a>.</em>

**Comment by David0z** [comment]
Currently the best Telegram downloader. Doesn't ommit files in comparison to the other ones. The file size limiter works well. I just can't wrap my head around the file type filter - I guess it needs a better documentation - although I didn't test it that much. Good job anyway

**Reply by DamianMorozov** [reply]
Thank you very much for such a high evaluation of my work. Things really are very bad with documentation. I would be glad if I could find a good specialist who could supplement my work by writing good documentation.

**Reply by DamianMorozov** [reply]
You can look here [Guide for filters settings](https://github.com/DamianMorozov/TgDownloader/blob/preview/GUIDES.md)
